### DISCUSSION TITLE
发现个好玩的

### DISCUSSION THREAD

**Discussion by originalFactor** [xfgryujk/blivechat#220] [通用]
头像现在能获取吗，（能获取就当我没说）

我发现B站`https://space.bilibili.com/<uid>`网页里就直接有明文的用户头像图片直链和昵称（）

昵称不用说到处都有，头像在`<head>`标签最后一行里，有个`<link rel="apple-touch-icon">`标签，它的`href`属性就是完整链接（在前面拼个`https:`就行）

**Comment by originalFactor** [comment]
甚至是原图（）甚至不用运行Js（）

**Comment by SSSSSSSan** [comment]
因为是伪静态
